tests/integration/conftest.py

```diff
@@ -133,7 +130,7 @@ def dummy_census_data(tmp_path_factory):
             "relation_to_household_head": [
                 random.choice(RELATIONS_TO_HOUSEHOLD_HEAD) for _ in range(num_rows)
             ],
-            "zipcode": [str(float(random.randint(1, 99999))) for _ in range(num_rows)],
+            "zipcode": [str(random.randint(1, 99999)) for _ in range(num_rows)],
```

Comment:
A followup PR will address this. Post-processing should save zipcodes and itins out as ints and not floats (they're currently coming in like "92823.0")